docs/apm/filters.asciidoc

```diff
@@ -6,49 +6,37 @@
 <titleabbrev>Filter data</titleabbrev>
 ++++
 
-APM provides two different ways you can filter your data within the APM App:
-
-* <<global-filters>>
-* <<contextual-filters>>
-
-[[global-filters]]
-==== Global filters
-
-Global filters are ways you can filter any and all data across the APM app.
-They are available in the Services, Transactions, Errors, Metrics, and Traces views,
-and any filter applied will persist as you move between pages.
+APM provides global filters, including a global time range filter and a service
+environment filter, so that you can filter your data within the APM app.
 
 [role="screenshot"]
 image::apm/images/global-filters.png[Global filters available in the APM app in Kibana]
 
-[float]
-===== Global time range
+Global filters are ways you can filter data across the APM app based on a specific
+time range or environment. They are available in the Services, Transactions, Errors,
+Metrics, and Traces views, and any filter applied will persist as you move between pages.
```

Comment:
Now that everything else has been removed, these two paragraphs seem to just be repeating each other. Maybe remove L9-10 and move 15-17 up?

Reply:
Good point. Thanks!